cmd/charmil/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	cmd := &cobra.Command{
+		Use:          "charmil",
+		Short:        "Commands for managing plugins",

```

Comment:
![Uploading Screenshot 2021-06-17 at 16.19.19.png…]()

Reply:
Outdated code (as mentioned in comment https://github.com/aerogear/charmil/pull/63#discussion_r653680593).

Have removed it now